addons/etcd/templates/cmpd-with-shell.yaml

```diff
@@ -0,0 +1,152 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: etcd-{{ .Values.image.tag.major3.minor56 }}
```

Comment:
Although it is not prohibited, it is not encouraged to write a separate cmpd for each version of the engine. In the design, a single cmpd should support multiple versions of the engine until the new version of the engine differs significantly enough that the same cmpd cannot be reused.